integration/ct_integration_test.go

```diff
@@ -306,6 +306,44 @@ func TestCTIntegration(t *testing.T) {
 		t.Errorf("\tuploaded:  %s", hex.EncodeToString(tbs))
 		t.Errorf("\tretrieved: %s", hex.EncodeToString(ts.PrecertEntry.TBSCertificate))
 	}
+
+	// Stage 14: get an inclusion proof for the precert.
+	fmt.Printf("Inclusion proof leaf @ %d -> root %d = ", precertSCT.Timestamp, sthN1.TreeSize)
+	// Calculate leaf hash =  SHA256(0x00 | tls-encode(MerkleTreeLeaf))
+	issuer, err := x509.ParseCertificate(prechain[1].Data)
+	if err != nil {
+		t.Fatalf("failed to parse issuer certificate: %v", err)
+	}
+	leaf = ct.MerkleTreeLeaf{
+		Version:  ct.V1,
+		LeafType: ct.TimestampedEntryLeafType,
+		TimestampedEntry: &ct.TimestampedEntry{
+			Timestamp: precertSCT.Timestamp,
+			EntryType: ct.PrecertLogEntryType,
+			PrecertEntry: &ct.PreCert{
+				IssuerKeyHash:  sha256.Sum256(issuer.RawSubjectPublicKeyInfo),
+				TBSCertificate: tbs,
+			},
+			Extensions: precertSCT.Extensions,
+		},
+	}
+	leafData, err := tls.Marshal(leaf)
+	if err != nil {
+		t.Fatalf("tls.Marshal(precertLeaf)=nil,%v", err)
```

Comment:
The reason I suggest the brackets is that you often want to log something like ", want %v" on the end of these (to make life easier for the soul who comes to debug this in 9 months' time), so bracketing disambiguates the string of comma-separated things a bit.
To me, the latter is easier to parse, YMMV...
"MyFunc(345) = 345, cheese is green, want 352, bananas are blue" 
vs. 
"MyFunc(345) = (345, cheese is green), want (352, bananas are blue)"